pkg/platform/runtime/target/target_test.go

```diff
@@ -15,7 +15,7 @@ func TestTrigger_IndicatesUsage(t *testing.T) {
 		},
 		{
 			"Import does not count as usage",
-			TriggerImport,
+			triggerUnknown,
 			false,
```

Comment:
This feels incorrect, the `TriggerImport` was right but given the changes we want to change the `false` to `true`.